pkg/util/kubernetes/autoscalers/datadogexternal.go

```diff
@@ -171,6 +191,15 @@ func (p *Processor) queryDatadogExternal(ddQueries []string, timeWindow time.Dur
 		}
 	}
 
+	// Update rateLimitsRemainingMin metric
+	updateMap := p.datadogClient.GetRateLimitStats()
+	queryLimits := updateMap[queryEndpoint]
+	newVal, err := strconv.Atoi(queryLimits.Remaining)
+	if err == nil {
+		getMinRemainingRequestsTracker().update(newVal)
+		rateLimitsRemainingMin.Set(float64(minRemainingRequestsTracker.val), queryEndpoint, le.JoinLeaderLabel)
```

Comment:
You should provide a `get()` on `minTracker` just to avoid leaking internal details

Reply:
thanks

updated.